src/components/ContestList.js

```diff
@@ -7,7 +7,7 @@ const ContestList = ({ contests, updateContestStatus }) => {
       {contests.map((contest) => (
         <ContestTile
           contest={contest}
-          key={contest.node.id}
+          key={contest.id}
```

Comment:
It doesn't seem necessary to do some of these plumbing changes.

```suggestion
          key={contest.node.id}
```

Reply:
It's important change actually because the object passer in contestList is now different